### DISCUSSION TITLE
List skipped updates at the end

### DISCUSSION THREAD

**Discussion by TopBanana123** [jojo2357/kiwix-zim-updater#28] [Ideas]
I have an idea for a feature where after updating, it will give a nice easily readable of updates skipped due to being too large, when using the -x flag. I set -x 10G to stop it trying to download Wikipedia for example, as its best to get that via torrent. But then I need to scroll up and try find all the ones it skipped. It would be nice if it could list them at the end so I could then manually go find the zim files

Great tool! Super handy

**Comment by jojo2357** [comment]
would it be sufficient if the log included notes on this? I just realized that it doesnt explain that it skipped one due to file sizing. I dont like the idea of putting a summary at the end because that would mean i have to save all the nuance of how the download went when it is much easier to just print it out as it comes.

**Reply by jojo2357** [reply]
So this is a small change that already has a workaround, so i will accumulate ideas for the next update (if your scroll wheel can last that long). 

As for the torrents, would there be something useful the script could do to help you download the larger archives via p2p? I suppose at minimum it could just fetch you magnets or torrents, but whats he maximum you would want it to do?

Just wondering since i dont (and cant) use p2p in my current situation.

**Comment by TopBanana123** [comment]
I think that would be great, really anywhere would work
